assets/js/translator.js

```diff
@@ -6,6 +6,7 @@ var curSrcLang, curDstLang;
 var recentSrcLangs = [], recentDstLangs = [];
 var droppedFile;
 var textTranslateRequest;
+var webpageTranslateRequest;
```

Comment:
Since there would only ever be one of text/webpage translation happening at the same time, do you think we could combine these?